internal/declarative/v2/renderer_raw_test.go

```diff
@@ -1,258 +0,0 @@
-package v2_test
```

Comment:
Why this test get removed?

Reply:
I removed this test because it was failing after I changed the raw renderer implementation. It was testing only the happy path and was too coupled to the original raw_renderer implementation. I think the manifest controller tests are sufficient.

Reply:
This test is only for render raw manifest, that's the purpose it is designed for, if it fails, then it means either your change introduced problems, or you need to adapter the test, but not completely delete it.